### PR TITLE
Seperate pr for audio 

### PR DESCRIPTION
**Pull request by tphan5901**
New settings for users to add audio and play audio files

### CODE REVIEW

File: js/effect/blur.js
Comment (Wei-1) [inline js/effect/blur.js:1]: Let's remove all changes from this file.

File: js/effect/effect-manager.js [hunk 1]
```diff
@@ -2,6 +2,19 @@
 
 function initEffect() {}
 
+
```
Comment (Wei-1) [inline js/effect/effect-manager.js:5]: remove this change

File: js/gui/gui-layout.js [hunk 2]
```diff
@@ -170,6 +170,64 @@ function createBGColorLayout(group) {
...
+        listItem.className = "music-item"
+        listItem.style.display = "flex"  
+        listItem.style.alignItems = "center"  
+    
+        //decode
+        const arrayBuffer = music.data 
+        const blob = new Blob([arrayBuffer], { type: 'audio/mp3' }) 
+        const url = URL.createObjectURL(blob) 
+
+        //assign url to new audio object
```
Comment (Wei-1) [inline js/gui/gui-layout.js:191]: I will suggest to change to code to this for synchronized style:
```

        //assign url to new audio object
        const audio = new Audio(url)  
        audio.controls = true   
        audio.style.height = "28.5px"
        listItem.appendChild(audio)
    
        //element
        const deleteBtn = document.createElement('button')
        deleteBtn.innerText = getL('Delete')
        deleteBtn.onclick = () => {
            const songId = songList[index].id
            songList.splice(index, 1)
            deleteSong(songId)
            updateMusicList()
        }
        listItem.appendChild(deleteBtn)

        const loopBtn = document.createElement('button')
        loopBtn.innerText = "⟳Loop"
        loopBtn.onclick = () => {
            audio.loop = !audio.loop
            loopBtn.innerText = audio.loop ? "Looping" : "⟳Loop"
        }
        listItem.appendChild(loopBtn)
        musicList.appendChild(listItem)
```

File: js/gui/gui-layout.js [hunk 3]
```diff
@@ -330,7 +392,7 @@ function createLayout() {
                         effectitem['enableEffect']();
                         itemdiv.style.display = "block";
                     } else {
-                        effectitem['disableEffect']();
+                        effectitem['disableEffect']();
```
Comment (Wei-1) [inline js/gui/gui-layout.js:384]: remove spaces after `;`

File: js/gui/gui-layout.js [hunk 4]
```diff
@@ -395,6 +457,40 @@ function createLayout() {
...
+    let songCounter = 0;
+    let musicboxbtn = document.getElementById("musicboxbutton")
+    musicboxbtn.innerHTML = getL("Music"); 
+    let musicbox = document.getElementById("musicbox") 
+    musicbox.innerHTML = ""   //
+    let musicbtn = document.createElement('input')
+    musicbtn.setAttribute("type", "file")
+    musicbtn.setAttribute("accept", ".mp3")
+    musicbox.appendChild(musicbtn)
+    musicbtn.onchange = function() {  //.onclick = () => {}
```
Comment (Wei-1) [inline js/gui/gui-layout.js]: remove `  //.onclick = () => {}`

File: js/gui/multi-language.js [hunk 5]
```diff
@@ -15,6 +15,16 @@ let languageBox = [{
     "en": 'Visual Effect',
     "zh": '視覺特效'
 }, {
+    "en": 'Music',
+    "zh": '', //
```
Comment (Wei-1) [inline js/gui/multi-language.js]: `"zh": '音樂'`

File: js/gui/multi-language.js [hunk 6]
```diff
@@ -15,6 +15,16 @@ let languageBox = [{
...
+    "jp": '音楽（ＢＧＭ）'
+},
+ {
+    "en": 'Delete',
+    "zh": '',
```
Comment (Wei-1) [inline js/gui/multi-language.js]: `"zh": '刪除'`

**Review by Wei-1 — COMMENTED** [review]
@tphan5901 thanks for the PR!
I like the changes, there are few css update that I would like to have!
Please update the code and let me know!

**Review by tphan5901 — COMMENTED** [review]
I generally had issues with the sizing of audio controls. I think your solution works best.

**Pull request merged**